service/init.go

```diff
@@ -621,10 +621,6 @@ func (amf *AMF) UpdateConfig(commChannel chan *protos.NetworkSliceResponse) bool
 	for rsp := range commChannel {
 		logger.GrpcLog.Infof("Received updateConfig in the amf app : %v", rsp)
 		var tai []models.Tai
-		var plmnList []*factory.PlmnSupportItem
```

Comment:
what would happen if response is nil. how its going to help?

Reply:
This line was simply removed because `plmnList` is not used anywhere. The problem was specifically:

```
if rsp.NetworkSlice == nil {
			return false
		}
```

Reply:
I added comment at wrong place. I understand plmnlist was not used. My question was about rsp.NetworkSlice.
but I see infinite for loop and program should wait there forever.